### PR TITLE
Fix samples

### PR DESCRIPTION
**Pull request by mshima**

### CODE REVIEW

**Comment by pascalgrimaud** [comment]
thanks @mshima - trusting you on this :)

**Comment by mshima** [comment]
@MathieuAA the fix to bug-tracker sample is correct?

**Comment by MathieuAA** [comment]
Looks okay yes

**Comment by DanielFran** [comment]
@mshima Not sure why you substitute pager with pagination?

**Comment by pascalgrimaud** [comment]
@DanielFran : `pager` has been removed and replaced by `pagination`. It's similar to https://github.com/jhipster/jdl-samples/pull/20

**Comment by DanielFran** [comment]
I never realised that pager was removed... But I did not use it! :P

File: bug-tracker.jh [hunk 1]
```diff
@@ -6,6 +6,12 @@ entity Label {
   label String required minlength(3)
 }
 
+application {
+  config {
+    prodDatabaseType postgresql,
+  }
```
Comment (pascalgrimaud) [inline bug-tracker.jh:12]: maybe missing `entities *` ?
Just tried it and there is no generated entities

Comment (mshima) [reply bug-tracker.jh:12]: I tried to revert my change and still doesn’t generate the entities.

And the application is generated at the parent folder.
@MathieuAA when using a jdl without `application {}` the application is generated at `../` is this the correct behavior?

Comment (MathieuAA) [reply bug-tracker.jh:12]: No. Can you check again? It happens when I'm not in the right folder sometimes...

Comment (mshima) [reply bug-tracker.jh:12]: Yep, don't know what happened.
`Error: The JDL object, the application's name and its the database type are mandatory.`

Comment (mshima) [reply bug-tracker.jh:12]: I think I had a `.yo-rc.json` at the parent folder:
`Just found a '.yo-rc.json' in a parent directory`

**Pull request merged**